tools/tools.go

```diff
@@ -338,22 +341,107 @@ func EnvmanClear(envstorePth string) error {
 	return nil
 }
 
-// EnvmanRun ...
+const enableSecretFilteringKey = "BITRISE_ENABLE_SECRET_FILTERING"
+
+// filteringEnabled returns true if enableSecretFilteringKey env presents with true value, false otherwise.
+func filteringEnabled(secrets []envmanModels.EnvironmentItemModel) bool {
+	for _, secret := range secrets {
+		key, value, err := secret.GetKeyValuePair()
+		if err != nil {
+			return false
+		}
+		if key == enableSecretFilteringKey && value == "true" {
+			return true
+		}
+	}
+	return false
+}
+
+// EnvmanRun runs a command through envman.
 func EnvmanRun(envstorePth, workDirPth string, cmdArgs []string, timeout time.Duration, secrets []envmanModels.EnvironmentItemModel) (int, error) {
 	logLevel := log.GetLevel().String()
 	args := []string{"--loglevel", logLevel, "--path", envstorePth, "run"}
 	args = append(args, cmdArgs...)
 
+	if !filteringEnabled(secrets) {
+		command := command.NewWithStandardOuts("envman", args...).SetStdin(os.Stdin).SetDir(workDirPth)
+
+		if timeout <= 0 {
+			exitCode, err := command.RunAndReturnExitCode()
+			if err != nil {
+				err = errors.WithStack(err)
+			}
+			return exitCode, err
```

Comment:
please incude this in the if above so:
```
if exitCode, err := command.RunAndReturnExitCode(); err != nil {
return exitCode, errors.WithStack(err)
}
```

or even better not to use any if, just 
```
exitCode, err := command.RunAndReturnExitCode()
return exitCode, errors.WithStack(err)
```

because errors.WithStack(err) returns nil if err is nil